compress.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/klauspost/compress/zstd"
+)
+
+func CompressFile(inFile string) (string, error) {
+	in, err := os.Open(inFile)
+	if err != nil {
+		return "", err
+	}
+	defer in.Close()
+
+	outFile := fmt.Sprintf("%s.zst", inFile)
+	out, err := os.OpenFile(outFile, os.O_WRONLY|os.O_CREATE|os.O_EXCL, 0600)
```

Comment:
It might be worth also using `os.O_TRUNC`, in case the file already exists (who knows what happens on the user's system).

Reply:
I guess you're right, good point about just erroring out.